test/io_test.py

```diff
@@ -0,0 +1,63 @@
+"""Unittests for ais.io"""
+
+import contextlib
+import sys
+
+import ais
+
+import pytest
```

Comment:
@schwehr I left `pytest` out of `install_requires` because it is only needed to run the unittests and has no impact on a users ability to use the library.